### PR TITLE
Documentation improvements.

### PR DESCRIPTION
**Pull request by tobiasake**
Adding some missing documentations.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: wiki/markdown/docker.md [hunk 1]
```diff
@@ -75,17 +73,18 @@ When Eiffel-Intelligence container is running on your local Docker host, Eiffel-
 
 
 Another option to configure Eiffel-Intelligence is to provide the application properties file into the container, which can be made in two ways:
-1. Put application.properties file in Tomcat Catalina config folder in container and run Eiffe-Intelligence:
-
-`docker run -p 8070:8080 --expose 8080 --volume /path/to/application.properties:/usr/local/tomcat/config/application.properties eiffel-intelligence:0.0.19`
 
-2. Put application.properties file in a different folder in container and tell EI where the application.properties is located in the container:
+**1** Put application.properties file in Tomcat Catalina config folder in container and run Eiffe-Intelligence:
```
Comment (e-pettersson-ericsson) [inline wiki/markdown/docker.md]: Eiffel-Intelligence

Comment (tobiasake) [reply wiki/markdown/docker.md]: fixed.

File: wiki/markdown/docker.md [hunk 2]
```diff
@@ -18,29 +18,26 @@ With the Docker image user can try-out the Eiffel-Intelligence on a Docker Host
...
-`docker run -p 8070:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG -e spring.data.mongodb.host=mongodb -e spring.data.mongodb.port=27017 eiffel-intelligence:1.0.1`
+    
+    docker run -p 8070:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG -e spring.data.mongodb.host=mongodb -e spring.data.mongodb.port=27017 eiffel-intelligence:1.0.1
 
 MongoDB, RabbitMq and other Eiffel-Intelligence required components need to running and configured via these application properties that is provided to the docker command above. See the application.properties file for all available/required properties:
-[application.properties](https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/resources/application.properties)
+[application.properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
 
 
 # Some info of all flags to this command
```
Comment (e-pettersson-ericsson) [inline wiki/markdown/docker.md]: How come this heading is larger than the previous ones? I feel it should be on the same level with '##' and the subheadings for it (Docker flags and Eiffel Intelligence Spring properties) should be '###'

Comment (tobiasake) [reply wiki/markdown/docker.md]: This header is headline for the two coming sub-headers which is about java properties flags and Docker flags. And "of" should be "about", will update.

**Pull request merged**